packages/p2p/stores/advertiser-page-store.js

```diff
@@ -12,15 +12,17 @@ export default class AdvertiserPageStore {
     }
 
     @observable active_index = 0;
-    @observable advertiser_info = {};
     @observable ad = null;
+    @observable advertiser_full_name = '';
```

Comment:
Should be `computed`.

Reply:
I think it would be better if we used one observable for both first name and last name together (full_name) instead of setting observables individually but let's do this. It will be future-proof, in case we ever want to use the names separately. Thanks Aaron! :)